revscoring/datasources/meta/timestamp.py

```diff
@@ -0,0 +1,30 @@
+"""
+These meta-datasources operate on :class:`revscoring.Datasource`'s that
+return `mwtypes.Timestamp` of the given string.
+
+.. autoclass:: revscoring.datasources.meta.timestamp.Timestamp
+"""
+import mwtypes
+
+from ..datasource import Datasource
+
+MW_REGISTRATION_EPOCH = '2006-01-01T00:00:00Z'
+
+
+class Timestamp(Datasource):
```

Comment:
All other meta datasources are lower case and do something a bit more general.  I wonder what is the advantage to this as opposed to:

```
my_datasource = Datasource("my_datasource", process=mwtypes.Timestamp, depends_on=[my_datasource])
```